.clang-tidy

```diff
@@ -129,7 +129,6 @@ Checks: '-*,
   modernize-replace-auto-ptr,
   modernize-replace-random-shuffle,
   modernize-shrink-to-fit,
-  modernize-unary-static-assert,
```

Comment:
Please add a TODO about re-enabling this check, so that we don't forget it's been disabled.

Reply:
Oh I missed this comment but I just created one reenabling it.